sources/packages/forms/src/form-definitions/confirmsstudentenrollment.json

```diff
@@ -1,4101 +1,3458 @@
-                                                    "dbIndex": false,
-                                                    "customDefaultValue": "",
-                                                    "calculateValue": "",
-                                                    "calculateServer": false,
-                                                    "widget": null,
-                                                    "validateOn": "change",
-                                                    "validate": {
-                                                        "required": false,
-                                                        "custom": "",
-                                                        "customPrivate": false,
-                                                        "strictDateValidation": false,
-                                                        "multiple": false,
-                                                        "unique": false
-                                                    },
-                                                    "allowCalculateOverride": false,
-                                                    "encrypted": false,
-                                                    "showCharCount": false,
-                                                    "showWordCount": false,
-                                                    "allowMultipleMasks": false,
-                                                    "id": "ejvwsf6",
-                                                    "hideOnChildrenHidden": false,
-                                                    "addons": []
-                                                }
-                                            ],
-                                            "width": 6,
-                                            "offset": 0,
-                                            "push": 0,
-                                            "pull": 0,
-                                            "size": "md",
-                                            "currentWidth": 6
-                                        }
-                                    ],
-                                    "key": "columns2",
-                                    "type": "columns",
-                                    "input": false,
-                                    "tableView": false,
-                                    "placeholder": "",
-                                    "prefix": "",
-                                    "customClass": "",
-                                    "suffix": "",
-                                    "multiple": false,
-                                    "defaultValue": null,
-                                    "protected": false,
-                                    "unique": false,
-                                    "persistent": false,
-                                    "hidden":
```

Comment:
please remove `1` from here and other fields too

Reply:
Not part of my Changes.